python-polars/downloads.py

```diff
@@ -0,0 +1,14 @@
+import requests
+import pathlib
+
+
+def download_file(file_url: str, local_file_path: pathlib.Path) -> None:
+    "Download a file from the URL and save it with the specified file name."
```

Comment:
Per convention, we should use a triple-quoted string for doc-strings:

```suggestion
    """Download a file from the URL and save it with the specified file name."""
```